Traceback (most recent call last):
  File "/workspace/ACCV_2024/PRCSL/src/main.py", line 303, in <module>
    main()
  File "/workspace/ACCV_2024/PRCSL/src/main.py", line 251, in main
    appr.train(t, trn_loader[t], val_loader[t])
  File "/workspace/ACCV_2024/PRCSL/src/approach/incremental_learning_der.py", line 56, in train
    self.train_loop(t, trn_loader, val_loader)
  File "/workspace/ACCV_2024/PRCSL/src/approach/der.py", line 68, in train_loop
    super().train_loop(t, trn_loader, val_loader)
  File "/workspace/ACCV_2024/PRCSL/src/approach/incremental_learning_der.py", line 73, in train_loop
    self.train_epoch(t, trn_loader)
  File "/workspace/ACCV_2024/PRCSL/src/approach/der.py", line 89, in train_epoch
    for i, (images, targets, no_aug_images) in enumerate(trn_loader):
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 435, in __next__
    data = self._next_data()
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 475, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 44, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 44, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/workspace/ACCV_2024/PRCSL/src/datasets/memory_dataset.py", line 37, in __getitem__
    print("x: ", x.shape)
AttributeError: 'Image' object has no attribute 'shape'
